uint8 requester_vin #vin asking for the data
uint8 target_vin #vin we want the data from
uint16[] data #the frames we are requesting